checkstyle/suppressions.xml

```diff
@@ -321,13 +321,13 @@
 
     <!-- group coordinator -->
     <suppress checks="CyclomaticComplexity"
+              files="(ConsumerGroupMember).java"/>
+    <suppress checks="ParameterNumber"
               files="(ConsumerGroupMember|GroupMetadataManager).java"/>
     <suppress checks="(NPathComplexity|MethodLength)"
               files="(GroupMetadataManager|ConsumerGroupTest|GroupMetadataManagerTest).java"/>
     <suppress checks="ClassFanOutComplexity"
               files="(GroupMetadataManager|GroupMetadataManagerTest).java"/>
-    <suppress checks="ParameterNumber"
-              files="(ConsumerGroupMember|GroupMetadataManager).java"/>
```

Comment:
Why are we moving this one?

Reply:
You had put a comment to remove it in the last PR

Reply:
Hum.. I asked in the other PR if the change was necessary: https://github.com/apache/kafka/pull/13998/files#r1272466045. You said that you removed it but it is still here. I am confused.

Reply:
sorry I did remove it, when I double checked in the new PR I thought this removal was what I did, my bad I'll remove it